internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
```

Comment:
To improve error handling, consider returning a more informative error message if loading the configuration fails. Instead of just passing the original error, you can wrap it with a contextual message for better debugging.